rest_framework/views.py

```diff
@@ -223,9 +232,9 @@ def get_exception_handler_context(self):
         """
         return {
             'view': self,
-            'args': getattr(self, 'args', ()),
-            'kwargs': getattr(self, 'kwargs', {}),
-            'request': getattr(self, 'request', None)
+            'args': self.args,
+            'kwargs': self.kwargs,
+            'request': self.request,
```

Comment:
This seems correct. There is never a case where the view won't have args/kwargs/request, right?

Reply:
Well, it'll lead to an non understandable traceback if we get an exception in the parser or whatever happens before the view is called.